<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlFaultdetails</class>
 <widget class="QWidget" name="CtrlFaultdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="CtrlFaultdetails_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_line1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>500</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_line1_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>500</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string>故障名称:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string>故障代码:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label2_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string>故障位置:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label2_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string>故障设备:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label_FName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label_FCode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label_FPos">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label_FDevice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label2_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string>故障描述:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CtrlFaultdetails_label_FDetail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
color: rgb(255, 255, 255);
font: 16px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CtrlFaultdetails_Button_quit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(0, 0, 0);
background-color: rgb(216, 216, 216);
border:2px solid black;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <zorder>CtrlFaultdetails_background</zorder>
  <zorder>CtrlFaultdetails_line1</zorder>
  <zorder>CtrlFaultdetails_line1_2</zorder>
  <zorder>CtrlFaultdetails_label1</zorder>
  <zorder>CtrlFaultdetails_label2</zorder>
  <zorder>CtrlFaultdetails_label2_2</zorder>
  <zorder>CtrlFaultdetails_label2_3</zorder>
  <zorder>CtrlFaultdetails_label_FName</zorder>
  <zorder>CtrlFaultdetails_label_FCode</zorder>
  <zorder>CtrlFaultdetails_label_FDevice</zorder>
  <zorder>CtrlFaultdetails_label2_4</zorder>
  <zorder>CtrlFaultdetails_label_FDetail</zorder>
  <zorder>CtrlFaultdetails_Button_quit</zorder>
  <zorder>CtrlFaultdetails_label_FPos</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
